edb/server/protocol/execute.pyx

```diff
@@ -438,6 +449,99 @@ async def execute(
     return data
 
 
+async def _convert_parameters(
+    dbv: dbview.DatabaseConnectionView,
+    server_param_conversions: list[dbstate.ServerParamConversion],
+    in_type_args: Optional[list[dbstate.Param]],
+    bind_args: bytes,
+) -> Optional[list[args_ser.ConvertedArg]]:
+    # If there are server param conversions, compute them now so that they are
+    # injected into the recoded bind args later.
+    tenant = dbv.tenant
```

Comment:
Unused `tenant`, and this comment feels weird in format with the next comment - maybe use docstring here?

```suggestion
    """
    If there are server param conversions, compute them now so that they are
    injected into the recoded bind args later.
    """
```